src/modal/Modal.svelte

```diff
@@ -147,12 +165,17 @@
       {:else if layout === Layout.download}
         <DownloadWallet
           store={getStoreVersionFromBrowser()}
-          isArgent={Boolean(selectedConnector && (selectedConnector?.id === "argentMobile" || selectedConnector?.id === "argentX"))}
+          isArgent={Boolean(
+            selectedConnector &&
+              (selectedConnector?.id === "argentMobile" ||
+                selectedConnector?.id === "argentX"),
+          )}
           storeLink={selectedWallet?.download}
-          extensionName={selectedWallet?.name === "Argent" ? "Argent X" : selectedConnector?.name}
+          extensionName={selectedWallet?.name === "Argent"
```

Comment:
`selectedWallet?.name === "Argent"`

Is this ok? Should it be `selectedWallet?.name === "Ready (formerly Argent)"`?